File: A's/py/723A.py
def main() -> None:
    x1, x2, x3 = map(int, input().split())

    # Sort the coordinates to find the median
    coordinates: list[int] = sorted([x1, x2, x3])
    median: int = coordinates[1]

    # Calculate the total distance
    total_distance: int = abs(x1 - median) + abs(x2 - median) + abs(x3 - median)

    print(total_distance)

if __name__ == "__main__":
    main()

File: A's/cpp/723A.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int x1, x2, x3;
    cin >> x1 >> x2 >> x3;

    // Sort the coordinates to find the median
    int coordinates[] = {x1, x2, x3};
    sort(coordinates, coordinates + 3);

    int median = coordinates[1];

    // Calculate the total distance
    int total_distance = abs(x1 - median) + abs(x2 - median) + abs(x3 - median);

    cout << total_distance << endl;

    return 0;
}